go.mod

```diff
@@ -88,17 +88,15 @@ require (
 	github.com/docker/go-metrics v0.0.1 // indirect
 	github.com/docker/go-units v0.4.0 // indirect
 	github.com/felixge/httpsnoop v1.0.2 // indirect
-	github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e // indirect
+	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da // indirect
 	github.com/golang/protobuf v1.5.2
-	// snappy: updated for go1.17 support
```

Comment:
Is this deliberate?

Reply:
`make vendor` automatically do this. Maybe we need a `replace` if we want to lock the snappy version.

Reply:
hmm. What's the difference?

Reply:
I believe this will be fixed by https://github.com/moby/buildkit/pull/2348.